docs: Update root README with new links and modules (#2559)
JetBrains/Exposed: README.md

| `exposed-migration`           | Provides utilities to support database schema migrations                                                                                                                        |
| `exposed-money`               | Extensions to support [`MonetaryAmount`](https://javamoney.github.io/apidocs/java.money/javax/money/MonetaryAmount.html) from the [JavaMoney API](https://javamoney.github.io/) |
| `exposed-spring-boot-starter` | A starter for [Spring Boot](https://spring.io/projects/spring-boot) to utilize Exposed as the ORM instead of [Hibernate](https://hibernate.org/)                                |
| `spring-transaction`          | Transaction manager that builds on top of Spring's standard transaction workflow                                                                                                |

## Samples using Exposed

Follow the [Getting Started with DSL tutorial](https://www.jetbrains.com/help/exposed/getting-started-with-exposed.html) for a quick start or check out the [samples](samples/README.md) for more in-depth projects.

## Documentation

> Please visit the [Exposed website](https://www.jetbrains.com/exposed/) for access to the following links, as well as samples, tutorials, and support:

Reviewer: It's not necessary to mention the Exposed website because it will only redirect the readers to the actual pages. And we can skip mentioning support in this section.

```suggestion
For complete documentation, samples, and tutorials, see the following links:
```

-   [Documentation](https://www.jetbrains.com/help/exposed/home.html)
-   [Migration Guide](https://www.jetbrains.com/help/exposed/migration-guide-1-0-0.html)
-   [Breaking changes](https://www.jetbrains.com/help/exposed/breaking-changes.html)

> ## Reporting Issues / Support

Reviewer: I'd say reporting issues is a form of contributing. So the first section can go under `## Contributing ### Reporting issues`. And the second section into a separate `## Support` chapter.

We encourage your feedback in any form, such as feature requests, bug reports, documentation updates, and questions.
Please use [our issue tracker](https://youtrack.jetbrains.com/issues/EXPOSED) to report any issues or to log new requests.
While issues are visible publicly, either creating a new issue or commenting on an existing one does require logging in to YouTrack.

> Do you have questions or want to join the project conversation at our [#exposed](https://kotlinlang.slack.com/messages/exposed/) Slack channel?

Reviewer: ```suggestion
Have questions or want to contribute to the discussion? Join us in the [#exposed](https://kotlinlang.slack.com/messages/exposed/) channel on the [Kotlin Slack](https://kotlinlang.slack.com/).
```

> Feel free to [request an invitation](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up) for the [kotlinlang slack](https://kotlinlang.slack.com/).

Reviewer: ```suggestion
If you're not a member yet, you can [request an invitation](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up).
```

## Contributing

> We also actively welcome your pull requests. However, linking your work to an [existing issue](https://youtrack.jetbrains.com/issues/EXPOSED) is preferred.

Reviewer: This can then go under `### Submitting pull requests`.

```suggestion
We actively welcome your pull requests and encourage you to link your work to an [existing issue](https://youtrack.jetbrains.com/issues/EXPOSED).
```

Reviewer: We should also add a link to the contribution guidelines: https://www.jetbrains.com/help/exposed/contributing.html#pull-requests
